### ISSUE TITLE
snjekyll fails due to old ruby version

### ISSUE THREAD

**Issue by cherylepatrick** [SuperNEMO-DBD/SuperNEMO-DBD.github.io#71]
Problem on both my macbook pro and MacBook air. Not run this in a while so I'm not sure exactly when it started being broken.

If I run `snjekyll serve` (or `update-gems`, which I hoped might fix the problem), it fails with the error `Gem::InstallError: ruby_dep requires Ruby version >= 2.2.5, ~> 2.2.
An error occurred while installing ruby_dep (1.5.0), and Bundler
cannot continue.
Make sure that `gem install ruby_dep -v '1.5.0'` succeeds before bundling.`

I also get a warning `Warning: the running version of Bundler (1.14.6) is older than the version that created the lockfile (1.16.1). We suggest you upgrade to the latest version of Bundler by running gem install bundler.`

I ran  <brew install ruby>, updated bundler, and ran `gem install ruby_dep -v '1.5.0'`:

```
Cheryls-MacBook-Pro:SuperNEMO-DBD.github.io cpatrick$ ruby --version
ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin16]
Cheryls-MacBook-Pro:SuperNEMO-DBD.github.io cpatrick$ bundle --version
Bundler version 1.16.1
```
My guess is that the snjekyll script isn't finding the new versions but I can't work out why it wouldn't be. I tried opening a new terminal window and that didn't fix it. Any ideas? Thanks in advance!

**Comment by drbenmorgan** [comment]
@cherylepatrick, I'll take a look!

**Comment by cherylepatrick** [comment]
Thanks @drbenmorgan - hopefully it is something simple!

**Comment by drbenmorgan** [comment]
@cherylepatrick, I'm not sure of the exact cause of this, but try the following:

```
$ ./snjekyll clean
... output, might see some warnings, but should complete
$ ./snjekyll setup
```

That should create a fresh set of gems/exes for use. I've confirmed this to work(*) with both Mac system ruby (2.3.3) and Homebrew'd ruby (2.5.1).

(*) May not account for any ordering issues between switching between ruby versions, bu `snjekyll clean` should always get you back to the start.

I've also submitted PR #72 - I don't think this affects/will change anything, but maybe worth holding off until that's merged.
